### ISSUE TITLE
Rewrite Layer constructor, get rid of variable argument count

### ISSUE THREAD

**Issue by naomai** [naomai/php-layers#1]
There are currently 3 ways of calling the constructor:
```PHP
(resource $img)
(int $w, int $h)
(int $w, int $h, int $x, int $y)
````
Perhaps the new layer object should only be created internally in Image class, with a single constructor format.

**Comment by naomai** [comment]
```
$watermarkLayer = GDW\Layer::createFromFile(__DIR__ . "/cheesymemz.png");
```
maybe make use of chained calls:
```
$watermarkLayer = $layersImg->newLayer()->importFile(__DIR__ . "/cheesymemz.png");
```

**Comment by naomai** [comment]
resolved 1a056dd08a963276281b0e6e3f080d8fa93b653b
